src/constants/enums/cmsGovernanceTeams.ts

```diff
@@ -2,17 +2,20 @@ const cmsGovernanceTeams = [
   {
     key: 'technicalReviewBoard',
     label: 'Technical Review Board (TRB)',
-    value: 'Technical Review Board'
+    value: 'Technical Review Board',
+    apiKey: 'trbCollaborator'
```

Comment:
Maybe this should be `collaboratorKey`. The key is specific to the collaborator in the API.